### PR TITLE
test: Added app with many tests

### PR DESCRIPTION
**Pull request by piotradamczyk5**
Relates to #1650

Tests app with many tests could be built.

It is needed for example to add IT in #1650

### CODE REVIEW

File: test_projects/android/ops.sh [hunk 1]
```diff
@@ -23,4 +23,8 @@ function cucumber_sample_app() {
   $DIR/../../flank-scripts/bash/flankScripts assemble android app --copy --generate --artifacts=buildCucumberSampleApp
 }
 
+function many_tests_apk() {
```
Comment (jan-gogo) [inline test_projects/android/ops.sh:26]: Do we have any good reasons for supporting shell scripts, now? Wasn't a `flank-scripts` a replacement for the shell? 

This is not a change request, I am just thinking about adding an issue for removing any `ops.sh`

**Review by jan-gogo — APPROVED** [review]

File: flank-scripts/src/main/kotlin/flank/scripts/ops/assemble/android/BuildManyTestsApk.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    if (copy) copyManyTestsApk()
+}
+
+private fun copyManyTestsApk() {
+    val outputDir = Paths.get(flankFixturesTmpPath, "apk", "app_many_tests.apk")
+
+    if (!outputDir.parent.toFile().exists()) Files.createDirectories(outputDir.parent)
+
+    val assembleDirectory = Paths.get(androidTestProjectsPath, "app_many_tests", "build", "outputs", "apk", "debug", "app_many_tests.apk")
```
Comment (jan-gogo) [inline flank-scripts/src/main/kotlin/flank/scripts/ops/assemble/android/BuildManyTestsApk.kt:27]:
```suggestion
    val assembleDirectory = Paths.get(androidTestProjectsPath, "app_many_tests", "build", "outputs", "apk", "debug", "app_many_tests-debug.apk")
```

File: test_projects/android/ops.sh (hunk 1 above)
Comment (piotradamczyk5) [reply test_projects/android/ops.sh:26]: `ops.sh` just run `flankScripts` with the proper command it was added for backward compatibility, but sure we could remove it

File: test_projects/android/app_many_tests/src/androidTest/java/com/example/app/many/tests/ManyTests.kt [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+    }
+
+    @Test
+    fun test58() {
+        assertEquals(true, "true".toBoolean())
+    }
+
+    @Test
+    fun test59() {
+        assertEquals(true, "true".toBoolean())
```
Comment (Sloox) [inline test_projects/android/app_many_tests/src/androidTest/java/com/example/app/many/tests/ManyTests.kt:305]: Welldone what a pain in the ....

**Review by Sloox — APPROVED** [review]

File: test_projects/android/ops.sh (hunk 1 above)
Comment (bootstraponline) [reply test_projects/android/ops.sh:26]: +1 to not having shell scripts if possible

**Pull request merged**